dsn_test.go

```diff
@@ -1133,6 +1150,15 @@ func TestDSN(t *testing.T) {
 			},
 			dsn: "u:p@a.b.c.snowflakecomputing.com:443?ocspFailOpen=true&region=b.c&tmpDirPath=%2Ftmp&validateDefaultParameters=true",
 		},
+		{
+			cfg: &Config{
+				User:                     "u",
+				Password:                 "p",
+				Account:                  "a.b.c",
+				DisableQueryContextCache: true,
```

Comment:
let's add a test to check if the default value is used

Reply:
Same - nearly all HTAP tests use default value.